packages/internal-test-helpers/lib/ember-dev/setup-qunit.ts

```diff
@@ -29,4 +29,24 @@ export default function setupQUnit({ runningProdBuild }: { runningProdBuild: boo
       callback(hooks);
     });
   };
+
+  let originalThrows = QUnit.assert.throws;
+
+  (QUnit.assert as any).throwsAsync = async function(
```

Comment:
Why do we need this? What does this do better than what `assert.rejects` does?

Reply:
Discussed this offline, we aligned the naming and the API, but something about our version of `reject` is still working better for the time being.